tests/test_propertyMapPlot.py

```diff
@@ -180,7 +180,7 @@ def test_PropertyMapTractAnalysisTask(self):
             self._validateFigureStructure(fig, binsCount, xlabel, zoomFactors)
 
             # Validate the RGB fractions of the figure.
-            self._validateRGBFractions(fig, expectedRGBFraction, rtol=1e-7)
+            self._validateRGBFractions(fig, expectedRGBFraction, rtol=1e-3)
```

Comment:
Is there a principled reason for picking this value, or is it just empirical?  A comment with the answer might be good.

Reply:
It is empirical. Added in the comment.